packages/debug/src/index.ts

```diff
@@ -1,88 +1,174 @@
-import debug from 'debug'
+/* eslint-disable no-var */
+import * as kleur from 'kleur/colors'
+import { bold } from 'kleur/colors'
 
-import { Debug, Debugger } from './types'
+const MAX_ARGS_HISTORY = 100
+const COLORS = ['green', 'yellow', 'blue', 'magenta', 'cyan', 'red']
 
-const MAX_LOGS = 100
+const argsHistory: [namespace: string, ...unknown[]][] = []
+let lastTimestamp = Date.now()
+let lastColor = 0
 
-const debugArgsHistory: any[] = []
+globalThis.DEBUG ??= process.env.DEBUG ?? ''
+globalThis.DEBUG_COLORS ??= process.env.DEBUG_COLORS ? process.env.DEBUG_COLORS === 'true' : true
 
-// Patch the Node.js logger to use `console.debug` or `console.log` (similar to
-// the browser logger) in the Edge Client.
-if (typeof process !== 'undefined' && typeof process.stderr?.write !== 'function') {
-  debug.log = console.debug ?? console.log
```

Comment:
But what about this @millsp? This was the previous behavior for the Edge Client and we used to send logs to stdout there? I'm not sure why 🤔

Reply:
Discussion continued here https://github.com/prisma/prisma/pull/23013#discussion_r1486815148